game.js

```diff
@@ -380,11 +387,22 @@ function main() {
   });
 
   const game = new Game({
-    width: 25,
-    height: 25,
-    squareSize: 20,
+    width: WIDTH,
+    height: HEIGHT,
+    squareSize: SQUARE_SIZE,
     players: [player1, player2],
   });
 
-  game.start();
+  const table = document.getElementById("table-stats");
+  table.width = WIDTH*SQUARE_SIZE;
+  table.height = HEIGHT*SQUARE_SIZE;
+
+  const ui = new UI({
```

Comment:
why does the UI take so many parameters? As a user of the UI i should just be able to construct it as easily as possible. Why do I need to know the playButton's element id?